sema/discovery/discovery.py

```
@@ -152,6 +166,35 @@
+        log.debug(f"found {len(parser.scripts)} scripts in the html file")
+        l_s_dict["scripts"] = len(parser.scripts)
+
+        for script in parser.scripts:
+            # parse the script and check if it is json-ld or turtle
+            # if so then add it to the triplestore
+            for script_type, script_content in script.items():
+                log.debug(f"found script of type {script_type}")
+                log.debug(f"script content: {script_content}")
+                if script_type in self.SUPPORTED_MIMETYPES:
+                    self._add_triples_from_text(
+                        script_content, script_type, resp_url
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)